processor/htmlproc/htmldoc/node_test.go

```diff
@@ -163,7 +163,10 @@ func TestFindNode(t *testing.T) {
 			node := htmldoc.FindNode(doc, test.tag)
 			var got strings.Builder
 			if node != nil {
-				html.Render(&got, node)
+				err = html.Render(&got, node)
```

Comment:
I believe it's more common to write:

```go
if err := html.Render(&got, node); err != nil {
    t.Fatal(err)
}
```

Same for other similar instances.

Reply:
Done Nx.